website/_includes/footer.html

```diff
@@ -1,8 +1,15 @@
-<footer class="site-footer">
-  <div class="container" align="center">
-    <p style="font-size: 14px; font-weight: 600"><a href="{{ site.baseurl }}/about">About</a>&nbsp;&nbsp;&nbsp;&nbsp; <a href="{{ site.baseurl }}/contact">Contact</a>&nbsp;&nbsp;&nbsp;&nbsp; <a href="https://github.com/intuit/katlas">GitHub</a>&nbsp;&nbsp;&nbsp;&nbsp; </p>
-    <p style="margin-top: 15px; font-size: 14px;">&#169; 2018. <a href="https://katlas.io">{{ site.author.name }}</a>. <a rel="license" href="https://www.apache.org/licenses/LICENSE-2.0">Apache License</a>.</p>
-  </div>
+<footer class="site-footer" >
+
```

Comment:
@kevinli11, do you intend for all the extra whitespace in these changes?

Reply:
@davemasselink : I don't think I need them.